sanghyeok/머쓱이보다-키-큰-사람.js

```diff
@@ -0,0 +1,9 @@
+function solution(array, height) {
+    var answer = 0;
+    for( i of array){
+        if(i > height) answer++ 
+        
+    }
```

Comment:
```suggestion
    let answer = 0;
    for(const i of array){
        if(i > height) answer++ 
        
    }
```
- var 키워드는 사용하지 않도록 해주세요.
- for 문 변수 i에 키워드 설정 바랍니다.

Reply:
넴